app/assets/stylesheets/desktop/user-card.scss

```diff
@@ -1,317 +1,253 @@
   transition: opacity 0.2s, transform 0.2s;
-
+  -webkit-transition: opacity 0.2s, -webkit-transform 0.2s;
   opacity: 0;
   @include transform(scale(0.9));
-
   &.show {
     opacity: 1;
     @include transform(scale(1));
   }
-
   &.fixed {
     position: fixed;
     z-index: z("composer", "content") + 1;
   }
-
   .card-content {
-    padding: 12px 12px 0 12px;
-    background: rgba($user_card_background, 0.85);
+    padding: 12px;
+    background: rgba($secondary, 0.85);
     margin-top: 80px;
-
     &:after {
       content: "";
       display: block;
       clear: both;
     }
-
     a.card-huge-avatar {
       outline: none;
     }
   }
-
-  .bio {
-    max-height: 150px;
-    overflow: auto;
-  }
-
   &.no-bg {
     min-height: 50px;
-
     .card-content {
       margin-top: 0;
     }
   }
-
-  .avatar-placeholder {
-    width: 120px;
-    height: 120px;
-    float: left;
-    padding-right: 10px;
+  .names {
+    flex: 1 1 auto;
+    margin-left: 0.75em;
+    span {
+      display: block;
+    }
   }
-
   h1 {
-    min-width: 120px;
-    font-size: $font-up-4;
+    margin: 0;
     line-height: $line-height-medium;
-    display: block;
-    max-width: 250px;
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
     a {
-      color: $user_card_primary;
+      color: $primary;
     }
     .d-icon {
       font-size: $font-down-1;
-      color: $user_card_primary;
+      color: $primary;
     }
   }
-
   h2 {
     font-size: $font-up-1;
-    line-height: $line-height-large;
     margin: 0;
     font-weight: normal;
-    display: block;
-    max-width: 250px;
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
     a {
-      color: $user_card_primary;
+      color: $primary;
     }
   }
-
   h3 {
+    display: inline;
+    margin-right: 0.5em;
     font-size: $font-0;
     font-weight: normal;
-    margin-top: 0;
-    color: scale-color($user_card_primary, $lightness: 20%);
-    overflow: hidden;
-    text-overflow: ellipsis;
+    color: scale-color($primary, $lightness: 20%);
+    .desc,
     a {
-      color: scale-color($user_card_background, $lightness: 50%);
+      color: scale-color($primary, $lightness: 35%);
     }
   }
-
-  .groups {
-    font-size: $font-down-1;
-    font-weight: normal;
-    margin-top: 0;
-    color: $user_card_background;
-
-    .group-link {
-      color: $user_card_background;
-    }
+  p {
+    margin: 0 0 5px 0;
   }
-
-  .metadata {
-    width: 100%;
-    clear: both;
-    padding-top: 10px;
-    padding-bottom: 10px;
-    h3 {
-      display: inline;
-      margin-right: 5px;
-      .desc,
-      a {
-        color: scale-color($user_card_primary, $lightness: 35%);
-      }
-    }
-    div {
-      display: inline;
-      color: scale-color($user_card_background, $lightness: 50%);
-      .group-link {
-        color: scale-color($user_card_background, $lightness: 50%);
-      }
+  .btn {
+    margin: 0 0 7px 0;
```

Comment:
This 7px makes me raise an eye brow :p can’t we rely on more standard margins ?

Reply:
I share your concern about this. This was the ruled used previously and I didn't want to change anything visually. There's a lot we can do to improve / standardize our margins / padding but that's probably something that needs to happen separately.  

I've changed it to `margin-bottom: 5px` for now.